app/src/main/java/de/tum/in/tumcampusapp/utils/Const.kt

```diff
@@ -32,9 +32,8 @@ object Const {
     const val API_HOSTNAME_NEW = "app.tum.de"
     const val STUDY_ROOMS_HOSTNAME = "www.devapp.it.tum.de"
 
-    const val CURRENT_CHAT_ROOM = "current_chat_room"
+    // TODO Remove or rename
```

Comment:
is `CHAT_ROOM_DISPLAY_NAME` still nessesary?

Reply:
No.